src/utils/sources-tree/treeOrder.js

```diff
@@ -146,6 +188,16 @@ export function createTreeNodeMatcher(
     return createTreeNodeMatcherWithDebuggeeHost(debuggeeHost);
   }
 
+  if (isBundler(part)) {
+    // Specialied matcher, when we are looking for angular bundler.
```

Comment:
Remove 'angular' since we are looking for bundlers in general

Reply:
Okay, I missed this